mods/ra/rules/vehicles.yaml

```diff
@@ -821,6 +812,9 @@ QTNK:
 	SelectionDecorations:
 	MadTank:
 		DeployedCondition: deployed
+    WithRangeCircle:
```

Comment:
Indentation need to be changes from spaces to tabs.

Reply:
fixed